latha_kandipati/week_01/thursday/GeometryFunctions/js/Geometry_Functions.js

```diff
@@ -0,0 +1,120 @@
+/*
+Part 1, Rectangle
+
+Given the following a rectangle object like the one below, write the following functions:
+
+isSquare - Returns whether the rectangle is a square or not
+area - Returns the area of the rectangle
+perimeter - Returns the perimeter of the rectangle
+var rectangle = {
+  length: 4,
+  width: 4
+};
+*/
+//Checks if the rectangle is square or not based on the properties passed.
+function isSquare(l, w)
+{
+  var square = false;
+  if (l === w)
+  {
+    square = true;;
+  }
+  return square;
+}
+
+//Calculates the rectangle area based on the given properties.
+function calculateRecArea(l, w)
+{
+  return (l*w);
+}
+
+//Calculates the perimeter of the rectangle
+function calcRecPerimeter(l, w)
+{
+  return (2*(l+w));
+}
+var recLength = 6;
+var recWidth = 10;
+console.log("Rectangle properties: Length: ",recLength+ " Width: ",recWidth);
+console.log("Is the rectangle a square: ",isSquare(recLength, recWidth));
+console.log("Area of the rectangle: ",calculateRecArea(recLength,recWidth));
+console.log("Permeter of the rectangle: ",calcRecPerimeter(recLength,recWidth));
```

Comment:
We wanted you to pass the whole `rectangle` object into your functions, to give you practice in accessing properties of an object... but I have a feeling you already know how to do this, so nevermind.